### PR TITLE
Try to fix a surprising error from the new target selection.

### PR DESCRIPTION
**Pull request by chandlerc**
Also add some extra debug logging to try to help diagnose the error.

### CODE REVIEW

File: scripts/target_determinator.py [hunk 1]
```diff
@@ -59,6 +60,10 @@ def filter_targets(bazel: Path, targets: str) -> str:
             return p.stdout
         except subprocess.CalledProcessError as err:
             log(err.stderr)
+            log("Query file:\n```")
+            with open(tmp.name) as f:
+                log(f.read())
+            log("```")
```
Comment (jonmeow) [inline scripts/target_determinator.py]:
```suggestion
            log("Full query file:\n```\n")
            with open(tmp.name) as f:
                log(f.read())
            log("```")
```

**Review by jonmeow — APPROVED** [review]
You're getting a small suggestion mainly due to the missing newline. :)

(you might also consider one on the close)

File: scripts/target_determinator.py (hunk 1 above)
Comment (chandlerc) [reply scripts/target_determinator.py]: Log uses `print` which I thought already adds a newline at the end?

Comment (chandlerc) [reply scripts/target_determinator.py]: I tweaked the log string, but the docs for `print` indicate it includes a newline at the end.

**Pull request merged**